implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -169,7 +172,19 @@ private String createCounterLine(Meter meter, Measurement measurement) {
     }
 
     Stream<String> toTimerLine(Timer meter) {
-        return toSummaryLine(meter, meter.takeSnapshot(), getBaseTimeUnit());
+        if (!(meter instanceof DynatraceSummarySnapshotSupport)) {
```

Comment:
How can this happen? If the registry will give you a `DynatraceTimer` in case of V2 and this class is V2 only, this can't get anything else than a `DynatraceTimer`, or can it?